### DISCUSSION TITLE
How to Accelerate Concurrent Query Efficiency

### DISCUSSION THREAD

**Discussion by lixiaoyong12** [databendlabs/databend#9634] [Q&A] [answered]
当前一个节点，单个线程查询大概2秒左右，4个线程并发查询需要8秒左右？应该如何才能加单机的并发查询效率呢（当前max_threads已经是最大了）？

**Comment by bohutang** [comment, marked as answer]
If the concurrency query is larger than the CPU number, you can try:
set global max_threads=1;

**Reply by lixiaoyong12** [reply]
CPU is 32 cores， and max_threads=32， 4 concurrent threads query cost 8 seconds，one thread query cost 1-2 seconds
